app/lib/actions/katello/content_view/incremental_updates.rb

```diff
@@ -67,7 +67,65 @@ def run
         end
 
         def humanized_name
-          _("Incremental Update")
+          total_count = total_counts(input)
+          _("Incremental Update of %s Content View Version(s) " % total_count[:content_view_count]) + content_output(total_count)
+        end
+
+        def total_counts(input)
+          total_count = {}
+          content_view_count = 0
+          if input[:version_outputs]
+            input[:version_outputs].map do |version_output|
+              content_view_count += 1
```

Comment:
Do we need to do the increment of content_view_count?

Instead, outside of loop would either of the following be sufficient:
    content_view_count = input[:version_outputs].length
or do we even need the variable:
    total_count[:content_view_count] = input[:version_outputs].length